### PR TITLE
ldak: new package at 5.1

### PR DESCRIPTION
**Pull request by njbooher**

### CODE REVIEW

File: var/spack/repos/builtin/packages/ldak/package.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ldak(Package):
+    """LDAK is a software package for analyzing GWAS data"""
+
+    homepage = "http://dougspeed.com/ldak/"
+    url      = "http://dougspeed.com/wp-content/uploads/source.zip"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/ldak/package.py:13]: It's a shame there isn't a URL that contains the version number, but I couldn't find one either.

File: var/spack/repos/builtin/packages/ldak/package.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    version('5.1', sha256='9a3fe2fafc7b68cc57a17748a64db66f76b13acbd5e9a538ede20a46447fcf4a')
+
+    variant('mkl', default=False, description='Use MKL')
+
+    depends_on('zlib')
+    depends_on('blas')
+    depends_on('lapack')
+    depends_on('intel-mkl', when='+mkl')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/ldak/package.py]:
```suggestion
    depends_on('mkl', when='+mkl')
```
Spack has a virtual provider `mkl` that is provided by both `intel-mkl` and `intel-parallel-studio+mkl`

File: var/spack/repos/builtin/packages/ldak/package.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    def install(self, spec, prefix):
+        if '+mkl' in spec:
+            make('ldak')
+            mkdirp(prefix.bin)
+            install('ldak', prefix.bin)
+        else:
+            make('ldak_slow')
+            mkdirp(prefix.bin)
+            install('ldak_slow', join_path(prefix.bin, 'ldak'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/ldak/package.py]:
```suggestion
            install('ldak_slow', prefix.bin.ldak)
```

**Review by adamjstewart — APPROVED** [review]

**Comment by njbooher** [comment]
Getting a weird error now while trying to install it locally

==> Error: Package 'flexiblas' not found.

**Comment by njbooher** [comment]
Nevermind. Was using the wrong checkout of the repo

**Pull request merged**